tasks/ftp.js

```diff
@@ -56,4 +56,55 @@ module.exports = function (grunt) {
 			done();
 		});
 	});
+
+	grunt.registerMultiTask('ftpGet', 'Download files from an FTP-server', function () {
+		var done = this.async();
+		var options = this.options();
+		var fileCount = 0;
+
+		if (options.host === undefined) {
+			throw new Error('`host` required');
+		}
+
+
+		eachAsync(this.files, function (el, i, next) {
+			// have to create a new connection for each file otherwise they conflict
+			var ftp = new JSFtp(options);
+
+			grunt.file.mkdir(path.dirname(el.dest));
+
+			var finalLocalPath = el.dest;
+			if(grunt.file.isDir(el.dest)) {
```

Comment:
This test is used for this config example :
```js
files: {
	'fixtureGet/fixture2.txt': 'ftp/fixture/fixture.txt',
	'<%= dirs.local %>/fixtureGet': 'ftp/fixture/fixture.txt'
}
```

If the dest param is a directory, we create a new file on path : "dest_dir/src_filename".
May be there is a better way to do this ?